includes/functions.php

```diff
@@ -340,6 +341,7 @@ function tsml_plugin_deactivation()
     if (post_type_exists('tsml_group')) {
         unregister_post_type('tsml_group');
     }
+    tsml_import_cron_check( false );
```

Comment:
On plugin deactivate, remove TSML cron jobs. Reminds me to check if we remove cron events for change detection.